sdk/instrumentation/bodyattribute/bodyattribute.go

```diff
@@ -44,3 +44,48 @@ func SetTruncatedEncodedBodyAttribute(attrName string, body []byte, bodyMaxSize
 	span.SetAttribute(fmt.Sprintf("%s.truncated", attrName), true)
 	span.SetAttribute(attrName+".base64", base64.RawStdEncoding.EncodeToString(truncatedBody))
 }
+
+// SetPossiblyTruncatedBodyAttribute sets the body as a span attribute.
+// When body is being truncated, or already truncated we also add a second attribute suffixed by `.truncated` to
+// make it clear to the user, body has been modified.
+func SetPossiblyTruncatedBodyAttribute(attrName string, body []byte, bodyMaxSize int, truncated bool, span sdk.Span) {
```

Comment:
Does this one need to have truncation logic? It can just be used when client code wants to handle truncation on its own in which case no truncation logic is needed so function can just be `setBodyAtttribute(attrName string, body []byte, truncated bool, span sdk.Span)`.

Reply:
updated and used the `SetBodyAtttribute` from `SetTruncatedBodyAttribute`